docs/source/guide/Docker_README.md

```diff
@@ -6,19 +6,21 @@ repository-service-tuf-worker is part of Repository Service for TUF (RSTUF)
 
 ## Getting Started
 
-These instructions will cover usage information and for the docker container
+These instructions will cover usage information and for the docker container.
 
 ## Prerequisities
 
 
 In order to run this container you'll need docker installed.
 
-Some required services:
+Other requirements include:
 
-* repository-service-tuf-api
-* Compatible Borker and Result Backend Service with
+* repository-service-tuf-api service
+* Compatible broker and result backend service with
   [Celery](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html).
   Recomended: [RabbitMQ](https://www.rabbitmq.com) or [Redis](https://redis.com)
+* A key that is available online. For more instructions read the
+  [Deployment documentation](https://repository-service-tuf.readthedocs.io/en/latest/guide/deployment/server.html).
```

Comment:
I updated the suggestion above, mentioning the new feature in the CLI to generate the Key.
It would be an excellent way to go.

Reply:
I am only worry that I cannot give a link to the exact place where we will document that in read-the-docs now when it's not yet merged.

Reply:
Because of it, I added it to the repository-service-tuf-cli guide.
There can be found when it is also merged.
You can also see that I update the feature [issue](https://github.com/vmware/repository-service-tuf/issues/207), which requires the new CLI key generation feature.